magento/pwa-studio
# [Doc] state management tutorial

## Description

Creates revises and publishes a tutorial about component state management.

## Related Issue

Closes PWA-305

## Acceptance 

Any developer

### Verification Stakeholders

Any developer

### Specification



### Verification Steps
1. Navigate to the `pwa-devdocs` directory: `cd pwa-devdocs`
2. Run the linter: `yarn lint src/tutorials/pwa-studio-fundamentals/manage-state/index.md`
3. Verify no linting issues
4. Run preview server: `yarn develop`
5. Navigate to the tutorials section
6. Verify new topic appears in the left nav and leads to the correct page

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
